yt/data_objects/octree_subset.py

```diff
@@ -455,8 +456,8 @@ def __init__(self, ind, block_slice):
     def __getitem__(self, key):
         bs = self.block_slice
         rv = bs.octree_subset[key][:,:,:,self.ind].T
-        if bs.octree_subset._block_reorder:
-            rv = rv.copy(order=bs.octree_subset._block_reorder)
+        if bs.octree_subset._block_order:
+            rv = rv.copy(order=bs.octree_subset._block_order)
```

Comment:
Would this be better written as
```python
         rv = np.require(
            bs.octree_subset[key][:,:,:,self.ind].T,
            requirements=bs.octree_subset._block_order)
```
which I believe does the same job but avoid unnecessary copies.
            
```

Reply:
I agree !
Moreover, I was going to suggest moving the transposition to after the closing parenthesis but I tested it and you version is actually more efficient by a factor 4 for the 100x100 array it tested this on.